### ISSUE TITLE
Add. 17,156, foll. 1–12 | https://bl.vuexistapps.us/manuscript/786 (Part1)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#1091]
### Problems with Manuscript Overview

- Date is missing. 

### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [X] Missing content

### Problems with Physical Description

- Please notice this in the main paragraph of Phys. Des.
"numbered
  Excerpt: ܝܛ
,
  Excerpt: ܟ
, and
  Excerpt: ܟܐ
,"

- Extra space after the word "Ornamentation" in "(Ornamentation )"
- The sentence in the ornamentation section reads, "These leaves are decorated with interlaced ornaments and figures of birds; e.g. foll. 4b, 5b, and 9a." Since it is dragged out of the body of Wright's entry,  the reference of "these leaves" is not clear in the online version. 

### Categories of Physical Description Problems

- [X] Whitespace
- [X] Typo(s)
- [X] Incorrect content
- [ ] Missing content

**Comment by mikey-tang16** [comment]
Resolved
- In PhysDesc, "quote" changed to "foreign" to get rid of the "excepts." Does this solution keep the texts together?

Unresolved
- As for "these leaves" I think it's fine to leave as is.
@wlpotter

**Comment by wlpotter** [comment]
Yes, changing quote to foreign should get rid of the "excerpts" label, and in theory also the formatting where these appear on their own line.

For the "these leaves" in the deco note, it's probably fine as is but maybe let's change it to be more concise: "Foll. 4b, 5b, and 9a are decorated with interlaced ornaments and figures of birds"

Also just checking if the changes you made are pushed? I don't see them on the fa2025-edits branch...

**Comment by mikey-tang16** [comment]
"these leaves" changed to Foll.

Pushed more changes to the branch, please let me know if they are on fa2025-edits.
